pkg/analyze/database_shared.go

```diff
@@ -35,14 +35,26 @@ func compareDatabaseConditionalToActual(conditional string, result *collect.Data
 		return false, errors.New("unable to parse postgres connected analyzer")
 
 	case "version":
+		//semver requires major.minor.patch format to successfully compare versions.
+		if compVer := strings.Split(parts[2], "."); len(compVer) == 2 {
```

Comment:
Can we make use of [ParseTolerant](https://godoc.org/github.com/blang/semver/v4#ParseTolerant) function here instead of doing all this manually?

Reply:
I don't know how could I have missed it. The only thing with that function is that it doesn't recognize 'x' as a version wildcard. Most algorithms take the x as 0 when comparing, so I used Replace() to replace 'x' for '0'. Maybe it is not entirely necessary when parsing the actual version, but I think it wont do any harm to left it there. I could remove it if you want.